redis/tests/support/mock_cluster.rs

```diff
@@ -121,6 +121,34 @@ pub fn respond_startup(name: &str, cmd: &[u8]) -> Result<(), RedisResult<Value>>
     }
 }
 
+pub fn respond_startup_two_nodes(name: &str, cmd: &[u8]) -> Result<(), RedisResult<Value>> {
```

Comment:
instead of this, maybe just pull https://github.com/redis-rs/redis-rs/pull/843/files#diff-ae2ede8b12dd85b78d0a380121b8b51caedf3f46452a564b18454dc8b5fe354c ? That way we won't have logic conflicts later.